### PR TITLE
Add examples for protobuf binary format

### PR DESCRIPTION
**Pull request by nishantwrp**
[Protocol Buffers](https://protobuf.dev) are Google’s language-neutral, platform-neutral, extensible mechanism for serializing structured data. One of the key features protocol buffers (or protobuf) is that it can encode the messages to binary format having really small sizes.

This example contains simple steps on how you can leverage the `bytea` field to store the Protobuf binary data and setup a YugabyteDB CDC Connector that streams the changes to a kafka topic.

Along with a consumer microservice that reads the messages from kafka and re-creates the protobuf message using binary data.

### CODE REVIEW

File: protobuf-binary-data/scripts/users.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        update_user()
+        return
+    elif options.delete:
+        delete_user()
+        return
+
+    print("No/Invalid operation specified")
+
+if __name__ == "__main__":
+    main()
```
Comment (fourpointfour) [inline protobuf-binary-data/scripts/users.py]: Add an extra line at the end of this file.

File: protobuf-binary-data/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## Example
+1. Start YugabyteDB
+    This can be a local instance as well as a universe running on Yugabyte Anywhere. All you need is the IP of the nodes where the tserver and master processes are running.
+    ```sh
+    export NODE=<IP-OF-YOUR-NODE>
+    export MASTERS=<MASTER-ADDRESSES>
+    ```
+
+2. Create a table
```
Comment (fourpointfour) [inline protobuf-binary-data/README.md]: ~~Create a table~~

File: protobuf-binary-data/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+4. Start the docker containers
+
+    ```sh
+    docker-compose up -d
+    ```
+
+5. Deploy the source connectors
+
+    We have created two connector configurations which use `avro` and `json` as kafka serialization formats respectively.
```
Comment (fourpointfour) [inline protobuf-binary-data/README.md]: nit: We have ~~created~~ two...

File: protobuf-binary-data/consumer/pom.xml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>protobuftest</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>19</maven.compiler.source>
+        <maven.compiler.target>19</maven.compiler.target>
```
Comment (fourpointfour) [inline protobuf-binary-data/consumer/pom.xml:13]: Can these not be run with Java 11? I mean, is the requirement such that JDK 19 is mandatory?

File: protobuf-binary-data/scripts/users.py (hunk 1 above)
Comment (nishantwrp) [reply protobuf-binary-data/scripts/users.py]: Done

File: protobuf-binary-data/README.md (hunk 2 above)
Comment (nishantwrp) [reply protobuf-binary-data/README.md]: done

File: protobuf-binary-data/README.md (hunk 3 above)
Comment (nishantwrp) [reply protobuf-binary-data/README.md]: done

File: protobuf-binary-data/consumer/pom.xml (hunk 4 above)
Comment (nishantwrp) [reply protobuf-binary-data/consumer/pom.xml:13]: I think it will. it's just that I have JDK 19 locally and IntelliJ set it to 19 automatically.

**Review by fourpointfour — APPROVED** [review]
It would be good to click on "Resolve conversation" for all the comments before merging if they have been addressed.

**Pull request merged**